content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,185 @@
+
+https://excalidraw.com/#json=5645858175451136,8w-G0ZXiOfRYAn7VWpANxw
+
+### 🇺🇳 Translations
+
+It was important for us early on to make sure that Excalidraw was translated into many languages so that it could be used all over the world. The [initial implementation](https://github.com/excalidraw/excalidraw/pull/638) was done by [fernandoalava](https://github.com/fernandoalava), while the automatic integration with our [Crowdin project](https://crowdin.com/project/excalidraw) was set up by [lipis](https://github.com/lipis). Support for right-to-left languages was [implemented](https://github.com/excalidraw/excalidraw/pull/1154) soon after by [j-f1](https://github.com/j-f1). To top it off, [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) designed new font glyphs for several non-latin languages. You can read more [about how we manage translations on the blog](/enabling-translations/)!
+
+### 📱 Mobile first
+
+Touch support and mobile-optimized layout [was](https://github.com/excalidraw/excalidraw/pull/787) [first](https://github.com/excalidraw/excalidraw/pull/788) [added](https://github.com/excalidraw/excalidraw/pull/790) by [j-f1](https://github.com/j-f1). This includes the creation of a toolbar for mobile devices that displays relevant controls while still leaving most of the screen free for the canvas.
+
+### 📚 Library
+
+The library was [first implemented](https://github.com/excalidraw/excalidraw/pull/1787) by [petehunt](https://github.com/petehunt). After adding support for exporting/importing the library, we eventually introduced a public directory where you can share yours. Visit [libraries.excalidraw.com](https://libraries.excalidraw.com) for more.
+
+https://twitter.com/dbs_sticky/status/1340349749086580736
+
+### 📊 Excalicharts
+
+While we aim to keep Excalidraw simple to use, sometimes we hide little easter eggs that you need to find out for yourself (or find some hints by following our [Twitter account](https://twitter.com/excalidraw)). For example, you can copy any two-column dataset from a spreadsheet, or comma separated values (CSV) from a text file, and paste them into Excalidraw to quickly produce a chart. The [first implementation](https://github.com/excalidraw/excalidraw/pull/1723) was done by [petehunt](https://github.com/petehunt) and several [improvements](https://github.com/excalidraw/excalidraw/pull/2495) were made by [lipis](https://github.com/lipis).
+
+### 🏹 Lines and Arrows
+
+Possibly the single most complex feature in Excalidraw, lines/arrows have come a long way since the beginning. Initially, we've only had two-point lines. Multi-point support was added by [gasimgasimzada](https://github.com/GasimGasimzada), with improvements by [dai-shi](https://github.com/dai-shi), line editing by [dwelle](https://github.com/dwelle), and by popular demand the arrow binding by [xixixao](https://github.com/xixixao).
```

Comment:
```suggestion
Possibly the single most complex feature in Excalidraw, lines/arrows have come a long way since the beginning. Initially, we've only had two-point lines. Multi-point support was added by [gasimgasimzada](https://github.com/GasimGasimzada), with improvements by [dai-shi](https://github.com/dai-shi), line editing by [dwelle](https://github.com/dwelle), arrowheads by [steveruizok](https://github.com/steveruizok), and by popular demand the arrow binding by [xixixao](https://github.com/xixixao).
```